ansible/pbTestScripts/testJDK.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mv /vagrant/pbTestScripts/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* ~
+export TEST_JDK_HOME=$(find ~ -maxdepth 1 -type d -name "*jdk8u*"|grep -v ".*jre.*")
```

Comment:
Any reason not to just set `TEST_JDK_HOME` to `/vagrant/pbTestScripts/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8*`?

Reply:
There's 2 directories in there- the JDK to test, and the JRE, which has the same name but `-jre` at the end